rktlet/runtime/helpers.go

```diff
@@ -341,6 +341,8 @@ func generateAppSandboxCommand(req *runtimeApi.RunPodSandboxRequest, uuidfile, s
 		}
 	}
 
+	cmd = append(cmd, "--insecure-options=seccomp")
```

Comment:
Could you add a comment in the code explaining why and your link to the systemd PR?

I think it's better to have the explanation in the code rather than having to read the git history.

Reply:
Sure

Reply:
Added the same explanation as a comment.